.github/workflows/merge-queue.yml

```
@@ -213,21 +248,24 @@ jobs:
       - name: Get manifest data
         id: manifest
+        if: ${{ needs.prepare.outputs.MQ_SHOULD_RUN_BUILD == 'true' && needs.prepare.outputs.DOCKER_CHUNKS && needs.prepare.outputs.DOCKER_CHUNKS != '[]' }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           node scripts/ci/docker/bootstrap.mjs
           node scripts/ci/docker/get-data.mjs
       - name: Update helm-values repository
         uses: ./.github/actions/update-helm-values
-        if: ${{ steps.manifest.outputs.MQ_HAS_OUTPUT == 'true' }}
+        if: ${{ steps.manifest.outputs.MQ_HAS_OUTPUT == 'true' }} && ${{ needs.prepare.outputs.MQ_SHOULD_RUN_BUILD == 'true' && needs.prepare.outputs.DOCKER_CHUNKS && needs.prepare.outputs.DOCKER_CHUNKS != '[]' }}
```

actionlint (1.7.4)

if: condition "${{ steps.manifest.outputs.MQ_HAS_OUTPUT == 'true' }} && ${{ needs.prepare.outputs.MQ_SHOULD_RUN_BUILD == 'true' && needs.prepare.outputs.DOCKER_CHUNKS && needs.prepare.outputs.DOCKER_CHUNKS != '[]' }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)